index.d.ts

```diff
@@ -0,0 +1,8 @@
+declare module 'resin-jwt' {
+	import Strategy from 'passport-jwt';
+
+	export function strategy(opts: {secret: string, apiKeys: {[key: string]: string}; apiHost: string; apiPort: string}): Strategy;
```

Comment:
It feels a bit weird to be defining objects within the function parameters, can we export these to named objects?

Reply:
Just to confirm I understand, are you after:

```ts
declare module 'resin-jwt' {
	import Strategy from 'passport-jwt';

	interface StrategyOpts {
		secret: string;
		apiKeys: {[key: string]: string};
		apiHost: string;
		apiPort: string;
	}

	export function strategy(StrategyOpts): Strategy;
	...
}
```

?

Reply:
Yeah :)

Reply:
Fixed